index.html

```diff
@@ -626,35 +646,115 @@ <h4>Script Type</h4>
           </pre>
         </section>
         <section>
-          <h4>Primary Language</h4>
-          <p>The <dfn>Primary Language</dfn> is a mandatory property of a <a>DAPT Script</a>
+          <h4>Default Language</h4>
+          <p>The <dfn>Default Language</dfn> is a mandatory property of a <a>DAPT Script</a>
             which represents the default language for the <a>Text</a> content of <a>Script Events</a>.
-            This language may be the <a>Original</a> language or a <a>Translation</a> language.
+            This language may be one of the <a>Original Languages</a> or a <a>Translation</a> language.
             When it represents a <a>Translation</a> language, it may be the final language
             for which a dubbing or audio description <a>script</a> is being prepared,
             called the <dfn>Target Recording Language</dfn> or it may be an intermediate, or pivot, language
-             used in the workflow.
+            used in the workflow.
           </p>
 
-            <p>The <a>Primary Language</a> is represented in a <a>DAPT Document</a> by the following structure and constraints:
+          <p>The <a>Default Language</a> is represented in a <a>DAPT Document</a> by the following structure and constraints:
             <ul>
               <li>the <code>xml:lang</code> attribute MUST be present on the <code>&lt;tt&gt;</code> element and its value MUST NOT be empty.</li>
             </ul>
           </p>
           <p class="note">All text content in a <a>DAPT Script</a> has a specified language.
-            When multiple languages are used, the <a>Primary Language</a> can correspond to the language of the majority of <a>Script Events</a>,
-            to the language being spoken for the longest duration, or to the language arbitrarily chosen by the author.</p>
-          <p class="note">Text content is marked either as being either in the <a>Original</a> language or
-            as being a <a>Translation</a> independently of the <a>Primary Language</a>, using the <a>Text Language Source</a> property.</p>
-          <aside class="example">An <a>Original Language Transcript</a> <a>script</a> is prepared for a video
+            When multiple languages are used, the <a>Default Language</a> can correspond to the language of the majority of <a>Script Events</a>,
+            to the language being spoken for the longest duration, or to a language arbitrarily chosen by the author.</p>
+          <p class="note">The language of text content is marked either as being either <a>Original</a>  or
+            <a>Translation</a> independently of the <a>Default Language</a>,
+            using the absence or presence of the <a>Text Language Source</a> property,
+            and its value.</p>
```

Comment:
This note feels redundant with other parts of the specification and since it does not directly related to "Default Language" I would remove it.

Reply:
Done in [01d26df](https://github.com/w3c/dapt/pull/179/commits/01d26dfa237f62aa96d98db149b0e3fe825bd630)